cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -386,3 +358,141 @@ func ValidateKubeconfigsForExternalCA(outDir string, cfg *kubeadmapi.InitConfigu
 	}
 	return nil
 }
+
+// KubeConfigs is a mapping between kubeconfig file name and spec
+type KubeConfigs map[string]*kubeConfigSpec
```

Comment:
There was already a `map[string]*kubeConfigSpec` in the original code, so I added an alias for it and attached a `visit` method.
I'd rather this be a slice, consistent with `certsphase.Certificates` but that would have required more changes in an already large PR.